### PR TITLE
updated build.sh file

### PR DESCRIPTION
**Pull request by theNerdFromSiliconValley**
updated build.sh file
1) User-friendly
2) Fixed code signing
3) Better DMG creation, User can save 24.99$

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,14 +1,27 @@
 #!/bin/bash
 echo "Build Script for eduVPN (and derivatives)"
+
+#Check if HomeBrew Installed
```
Comment (fkooman) [inline build.sh]: as far as I know, installing homebrew requires root permissions. Maybe the script could just stop and ask the user to manually install homebrew before running the script again. The build script can be run as a normal user, right?

File: build.sh [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+  ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+fi
+
+
 # Check if the Carthage is installed
 if ! [ -x "$(command -v carthage)" ]; then
-  echo 'Error: Carthage is not installed.' >&2
+  echo 'Carthage is not installed. Installing Carthage' >&2
+  brew install carthage
   python -mwebbrowser https://github.com/Carthage/Carthage
```
Comment (fkooman) [inline build.sh]: The browser does not need to be opened anymore if you use homebrew to install carthage, right?

File: build.sh [hunk 3]
```diff
@@ -23,30 +36,30 @@ echo ""
...
+read -p "Enter Team Name: " CUSTOMTEAMNAME
 fi
 
-#Simple TeamID Validation. Apple Team ID always consists of 10 Character
-
+#Simple TeamID Validation. Apple Team ID always consists of 10 Characters
 if  ! [ "${#CUSTOMTEAMID}" == 10  ]
 then
 echo "Error: Team ID is not valid"
+exit 0
```
Comment (fkooman) [inline build.sh]: Typically you exit with a non-zero exit code to indicate failure.

File: build.sh [hunk 4]
```diff
@@ -99,15 +112,38 @@ UP=$(find $FILENAME -name "*.up.*.sh" -print)
...
+read -p "Create another disk image (requires DropDMG license) (y/n)?" choice
 case "$choice" in
   y|Y ) ;;
   n|N ) exit 0;;
   * ) echo "Invalid response."; exit 0;;
 esac
 
 echo ""
-echo "Creating a disk image"
+echo "Creating a disk image using DropDMG"
```
Comment (fkooman) [inline build.sh]: if createdmg works, dropdmg can probably be fully removed.

File: build.sh (hunk 1 above)
Comment (theNerdFromSiliconValley) [reply build.sh]: HI François Boss,
Ok, I will ask user to install homebrew manually 
.
Yes the build script can be run as normal user

File: build.sh (hunk 3 above)
Comment (theNerdFromSiliconValley) [reply build.sh]: HI François Boss,
Ok. I will put 1 here

File: build.sh (hunk 4 above)
Comment (theNerdFromSiliconValley) [reply build.sh]: HI François Boss,
Ok I am removing dropDMG completely

**Pull request closed without merging**